scripts/solar_system_screensaver.ssc

```diff
@@ -1515,7 +1515,8 @@ core.selectObjectByName("Moon", pointer = true);
 StelMovementMgr.setFlagTracking(true);
 
 }
-
var pick = pick *( Math.floor(Math.random()*9)+1);
+
+var pick = pick *( Math.floor(Math.random()*9)+1);

```

Comment:
Not sure where these whitespace fixes came from, weird?

Reply:
I also wondered. Some syntax checker? I'd prefer to not add empty lines here.